plugins/module_utils/network/iosxr/config/lldp_global/lldp_global.py

```diff
@@ -112,7 +135,7 @@ def set_state(self, want, have):
                   to the desired configuration
         """
         state = self._module.params["state"]
-        if state in ("merged", "replaced") and not want:
+        if state in ("merged", "replaced", "rendered") and not want:
```

Comment:
```suggestion
        if state in ("merged", "replaced", "rendered", "overridden") and not want:
```

Reply:
This module does not have overridden state